packages/node/test/integration/plugins/operator/OperatorPlugin.test.ts

```diff
@@ -93,4 +93,31 @@ describe('OperatorPlugin', () => {
             await createBroker(config)
         }).rejects.toThrow('Plugin operator doesn\'t support client config value "false" in network.node.acceptProxyConnections')
     })
+    
+    it('Accepts OperatorDiscoveryRequests', async () => {
+        const client = createClient(await fetchPrivateKeyWithGas())
+        const stream = await createTestStream(client, module)
+
+        const sponsorer = await generateWalletWithGasAndTokens()
+        const sponsorship1 = await deploySponsorshipContract({ streamId: stream.id, deployer: sponsorer })
+        await sponsor(sponsorer, await sponsorship1.getAddress(), 10000)
+        await delegate(operatorWallet, await operatorContract.getAddress(), 10000)
+        await stake(operatorContract, await sponsorship1.getAddress(), 10000)
+
+        broker = await startBroker({
+            privateKey: brokerWallet.privateKey,
+            extraPlugins: {
+                operator: {
+                    operatorContractAddress: await operatorContract.getAddress(),
+                }
+            }
+        })
+        await waitForCondition(async () => (await broker.getStreamrClient().getSubscriptions(stream.id)).length > 0)
+        // Ensure that heartbeat has been sent (setting heartbeatUpdateIntervalInMs lower did not help)
+        await wait(10000)
```

Comment:
Do we actually need to wait for a heartbeat to be sent here since there is only one broker?

Reply:
At the moment the node does not even know itself before it sees its own heartbeat. I didn't want to change that as I don't know if it's intentional or not

Reply:
To not have to rely on a timer, how about  putting this function in the beginning of the file
```ts
async function waitForHeartbeatMessage(operatorContractAddress: EthereumAddress): Promise<void> {
        const client = createClient(fastPrivateKey())
        const sub = await client.subscribe(formCoordinationStreamId(operatorContractAddress))
        await collect(sub, 1)
        await client?.destroy()
    }
```

and calling it here?